src/target/riscv/riscv-011_reg.h

```diff
@@ -0,0 +1,17 @@
+#ifndef OPENOCD_TARGET_RISCV_RISCV_REG_011_H
+#define OPENOCD_TARGET_RISCV_RISCV_REG_011_H
+
+#include "target/target.h"
+
+/**
+ * This file describes additional register cache interface available to the
+ * RISC-V Debug Specification v0.11 targets.
+ */
+
+/**
+ * Initialize register cache. After this function all registers can be
+ * safely accessed via functions described here and in `riscv_reg.h`.
+ */
+int riscv011_init_registers(struct target *target);
```

Comment:
I would like to propose a unified naming convention for the files and "public" functions provided by them:

For `riscv-011_<name>.h/c`, all the functions would be called `riscv011_reg_*`.

In this case, it could be:

```suggestion
int riscv011_reg_init_registers(struct target *target);
```
or
```suggestion
int riscv011_reg_init(struct target *target);
```
or similar.

What is your opinion on this?

Reply:
IMHO, it's a great suggestion and will significantly improve readability, thanks!
Addressed.